### DISCUSSION TITLE
Actividad 1 - Grupo 1

### DISCUSSION THREAD

**Discussion by germansalinas1994** [frlp-utn-ingsoft/catedra#93] [Show and tell]
https://github.com/Fedebravo12/vetsoft

**Comment by ivanduva** [comment]
Buenas!
La actividad está muy bien resuelta, queda aprobada.

- La app funciona y los tests pasan.
- Valoramos que hayan estandarizado los nombres de branches.
- Los conflictos están resueltos de buena manera y prolijo.
- Los commits están bastante bien, salvo algunas excepciones que estaría bueno que revisen para la próxima. Por ejemplo, este caso en donde el commit no es muy atómico: https://github.com/Fedebravo12/vetsoft/commit/3dd3f8048dae4e1b7ce756c44e38c1144d31566c

Buen trabajo!
Saludos
